mobile/src/components/mentions/EntitySuggestions.tsx

```
@@ -0,0 +1,304 @@
+
+  const activeEntry = triggerEntries.find(([, data]) => data?.keyword !== undefined);
+
+  if (!activeEntry || !activeEntry[1]) {
+    return null;
+  }
+
+  const [activeKey, activeData] = activeEntry;
+  const keyword = activeData.keyword ?? '';
+  const trimmedKeyword = keyword.trim();
+
+  const baseSuggestions: SuggestionItem[] = useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)